icons/icons.json

```diff
@@ -949,6 +949,22 @@
 				"name": "Git Blame",
 				"scope": "text.git.blame"
 			},
+			{
+				"extensions": [
+					"git-blame-ignore-revs",
+					".git-blame-ignore-revs"
```

Comment:
Those extensions could probably just be added to above `Git Blame` alias.

Reply:
Are you sure? These have a different syntax than git blames.

Reply:
Both alias types are based on plain text, currently. Thus it wouldn't make a difference.

If there's a package with syntax definitions for one or both files, we shouldn't use aliases here.

Reply:
Let's say I have a package (I actually do) that defines the following:

```sublime-syntax
%YAML 1.2
---
name: Git Blame Ignore Revisions
scope: text.git.blame.ignorerevs
version: 2

file_extensions:
  - git-blame-ignore-revs     # *.git-blame-ignore-revs

hidden_file_extensions:
  - .git-blame-ignore-revs    # ~/.git-blame-ignore-revs

contexts: ...
```

What happens then? I'm not sure I understand the ramifications.

Reply:
Your syntax would be overridden by the alias proposed in this PR.

Why?

A file icon creates a dummy syntax definition in _Packages/zzz A File Icon zzz_ for each defined `alias` and each missing `syntax`.

Upon start of ST if a real `syntax` for a certain scope is found, the dummy is deleted. That won't happen for `alias`.

> `alias` === all syntaxes in `"aliases": []` key
> `syntax` === all syntaxes in `"syntaxes": []` key

Aliases are mainly used to create a dummy syntax with special scope for an existing syntax, just for the purpose to assign special icon.

---

That said, if there's a chance for a 3rd-party package, you'd probably just want to move those definitions to `"syntaxes": []`.

Reply:
The distinction between `alias` and `syntax` is required only for backward compatibility with ST3.

As we can't scan for existing syntaxes easily in ST3, dummy syntaxes are only created for aliases. This means syntax packages must be present/installed for icons to be displayed.

Reply:
With sublimehq/Packages#3718 merged there's no need for this PR to provide the `text.git.blame.ignorerevs` scope - I've removed the related code.